src/main/java/com/volmit/iris/util/decree/virtual/VirtualDecreeCommand.java

```diff
@@ -411,7 +411,7 @@ public boolean invoke(VolmitSender sender, KList<String> args, KList<Integer> sk
         if(isStudio() && !IrisSettings.get().isStudio())
         {
             sender.sendMessage(C.RED + "To use Iris Studio Commands, please enable studio in Iris/settings.json (settings auto-reload)");
-            return false;
+            return true; // Must still return true because command exists but is not enabled
```

Comment:
RETURN FALSE, THATS NOT WHAT FALSE OR TRUE MEANS

Return false so the tree searcher can continue to look for other commands that may match the input

Reply:
Six